public/script.js

```diff
@@ -0,0 +1,123 @@
+//tmp_list = ['apple', 'orange', 'grape', 'strawberry', 'apples'];//テスト用
+let created_sentence = [];
+
+const container = document.getElementById('container');
+
+window.addEventListener('load', (event) => {
+    // ウィンドウが読み込まれた時の処理
+    fetch('/api/cocktail/all')
+        .then(response => {
+            if (!response.ok) {
+                throw new Error('Network response was not ok');
+            }
+            //return response.json(); // JSON形式のレスポンスをパース
+            console.log(response);
+        })
+        .then(data => {
+            console.log('Received data:', data); // 取得したデータをコンソールに出力
+            // ここで、データを使った他の処理を実行することができます
+        })
+        .catch(error => {
+            console.error('Error fetching data:', error); // エラーハンドリング
+        });
+});
+
+
+function buttonclick(obj){
+    let text = '';
+    obj.childNodes.forEach(node => {
+        if(node.nodeType === node.TEXT_NODE){
+            text += node.textContent.trim();
+        }
+    });
+    const b_text = text;//クリックしたボタンの文字
```

Comment:
```suggestion
    const selectedCocktailMessage = text;//クリックしたボタンの文字
```
基本的に、javascriptの変数はlowerCamelCaseで書きます。
また、b_textだと何の変数か、将来の自分を含め分からなくなると思うので上記のようなのはどうでしょうか？ 🤔